src/lib/util/env.ts

```diff
@@ -1,6 +1,7 @@
 export const env = {
-  rendererUrl: import.meta.env.MERMAID_RENDERER_URL ?? 'https://mermaid.ink',
-  krokiRendererUrl: import.meta.env.MERMAID_KROKI_RENDERER_URL ?? 'https://kroki.io',
+  rendererUrl: import.meta.env.MERMAID_RENDERER_URL ?? '',
+  krokiRendererUrl: import.meta.env.MERMAID_KROKI_RENDERER_URL ?? '',
   analyticsUrl: import.meta.env.MERMAID_ANALYTICS_URL ?? '',
-  domain: import.meta.env.MERMAID_DOMAIN ?? 'mermaid.live'
+  domain: import.meta.env.MERMAID_DOMAIN ?? '',
+  isEnabledMermaidChartLinks: import.meta.env.MERMAID_IS_ENABLED_MERMAID_CHART_LINKS == 'True'
```

Comment:
```suggestion
  isEnabledMermaidChartLinks: import.meta.env.MERMAID_IS_ENABLED_MERMAID_CHART_LINKS == 'true'
```

To align with JS `true`/`false`.